src/action.ts

```diff
@@ -281,6 +285,19 @@ class Action {
     return dir
   }
 
+  private rmdir(outputsDir: string): string {
+    core.debug(`Removing directory ${outputsDir} after action finishes.`)
+    if (fs.existsSync(outputsDir)) {
+      fs.readdirSync(outputsDir)
+      if (fs.lstatSync(outputsDir).isDirectory()) {
+        fs.rmdirSync(outputsDir, { recursive: true })
+      } else {
+        fs.unlinkSync(outputsDir)
+      }
+    }
+    return outputsDir
+  }
```

Comment:
Since the function name is already `rmdir`, I'd skip the directory check. In fact I wouldn't create a new function if we're not catching or doing anything additional. The way is done right now, I'd just call `fs.rmdirSync` whenever is needed.

However, if `fs.rmdirSync` throws an Error in the event that the target directory does not exist, then I'd keep this function and catch that Error to avoid breaking the action execution.